plugins/image2/plugin.js

```diff
@@ -378,8 +378,10 @@
 						width: image.getAttribute( 'width' ) || '',
 						height: image.getAttribute( 'height' ) || '',
 
-						// Lock ratio is on by default (https://dev.ckeditor.com/ticket/10833).
-						lock: this.ready ? helpers.checkHasNaturalRatio( image ) : true
+						// Lock ratio should respect the value of the config.image2_defaultLockRatio.
+						// If the variable is not set, then it defaults to true (#5219).
+						lock: editor.config.image2_defaultLockRatio !== undefined ?
+							editor.config.image2_defaultLockRatio : true
```

Comment:
We changed the original logic for that widget, but I'm not able to reproduce https://dev.ckeditor.com/ticket/10833. I suppose that's the same issue with using `checkHasNaturalRatio` so inability to properly test that method in modern browsers. WDYT about keeping the old behavior untouched just to make sure we won't introduce any regression with that addition?